app/service/elasticsearch_service.py

```diff
@@ -34,3 +34,9 @@ def execute_search_query(self, search_query: SearchQuery):
             },
         )["hits"]["hits"]
         return [pattern["_source"] for pattern in result]
+
+    def insert_document(self, paragraph: str, header: str, labels: list):
```

Comment:
Also let's create a DAO (data access object) for this class with to_json() method that can be used in line 41